app/views/layouts/application.html.erb

```diff
@@ -45,6 +45,7 @@
 
     <%= javascript_pack_tag 'application' %>
 
+    <%= stylesheet_pack_tag 'application' %>
```

Comment:
It seems that Webpacker does require NPM packages' styles in the `development` environment but requires this explicit tag in the `production` environment. The reference here - https://github.com/rails/webpacker/issues/2059.

With this, not sure if the explicit import of the CSS file in the above React component is even needed with this change or not. It doesn't hard if we keep both the changes though.